src/blinker/base.py

```diff
@@ -209,6 +210,19 @@ def connected_to(self, receiver, sender=ANY):
         else:
             self.disconnect(receiver)
 
+    @contextmanager
+    def muted(self):
```

Comment:
Could you please add this to the docs as well?

Reply:
Yep, added some simple docs.